integration-testing/smokeTest/build.gradle.kts

```diff
@@ -78,12 +78,4 @@ kotlin {
             jvmTarget.set(JvmTarget.JVM_1_8)
         }
     }
-}
-
-// Drop this configuration when the Node.JS version in KGP will support wasm gc milestone 4
-// check it here:
-// https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/js/nodejs/NodeJsRootExtension.kt
-rootProject.extensions.findByType(NodeJsRootExtension::class.java)?.apply {
-    nodeVersion = "21.0.0-v8-canary202309167e82ab1fa2"
-    nodeDownloadBaseUrl = "https://nodejs.org/download/v8-canary"
 }
```

Comment:
new line? (to avoid github from complaining, and for the symmetry of the universe, but I understand if that's not the policy here)

Reply:
Done